src/pybind/mgr/cephadm/inventory.py

```diff
@@ -517,6 +522,22 @@ def load(self):
                     host, e))
                 pass
 
+    def _stray_host_cache_entry(self, host: str) -> str:
+        # return true if entry is stray and must be removed
+        # need to check for normal host cache entry and device entry
+        # <hostname>   OR   <hostname>.device.<integer>
```

Comment:
From the doc it seems that this function returns a boolean but in fact returns a string. BTW: wouldn't be better to return an Enum instead of a string?

Reply:
changed to return an enum and updated code comment